node/src/GlideClient.ts

```diff
@@ -371,6 +372,37 @@ export class GlideClient extends BaseClient {
         return this.createWritePromise(createTime());
     }
 
+    /**
+     * Copies the value stored at the `source` to the `destination` key on `destinationDB`. When `replace`
+     * is true, removes the `destination` key first if it already exists, otherwise performs no action.
+     *
+     * See https://valkey.io/commands/copy/ for more details.
+     *
+     * since Valkey version 6.2.0.
+     *
+     * @param source - The key to the source value.
+     * @param destination - The key where the value should be copied to.
+     * @param destinationDB - The alternative logical database index for the destination key.
+     * @param replace - If the destination key should be removed before copying the value to it.
+     * @returns `true` if `source` was copied, `false` if the `source` was not copied.
+     *
+     * @example
+     * ```typescript
+     * const result = await client.copy("set1", "set2", 1, false);
+     * console.log(result); // Output: true - "set1" was copied to "set2".
+     * ```
+     */
+    public async copyDB(
```

Comment:
Is there a reason why we're using copyDB as the name instead of copy? If its because copy is incompatible with the copy implementation in BaseClient.ts, this problem would be fixed if we moved that copy implementation to GlideClusterClient.ts

Reply:
Database is only supported in standalone not cluster. So that is why I don't want to expose optional `destinationDB`  in cluster mode. Yes, if I use the same method name with different argument, it will introduce the error.

Reply:
So move it to standalone and cluster client classes and you won't have collision